elasticsearch5/src/main/java/org/vertexium/elasticsearch5/ElasticsearchEdge.java

```diff
@@ -0,0 +1,83 @@
+package org.vertexium.elasticsearch5;
+
+import org.vertexium.*;
+import org.vertexium.mutation.ExistingEdgeMutation;
+
+public class ElasticsearchEdge extends ElasticsearchElement implements Edge {
+    private String label;
+    private String inVertexId;
+    private String outVertexId;
+
+    public ElasticsearchEdge(
+            Graph graph,
+            String id,
+            String label,
+            String inVertexId,
+            String outVertexId,
+            FetchHints fetchHints,
+            Authorizations authorizations
+    ) {
+        super(graph, id, fetchHints, authorizations);
+        this.label = label;
+        this.inVertexId = inVertexId;
+        this.outVertexId = outVertexId;
+    }
+
+    @Override
+    public String getLabel() {
+        return label;
+    }
+
+    @Override
+    public String getVertexId(Direction direction) {
+        if (direction.equals(Direction.IN)) {
+            return inVertexId;
+        } else if (direction.equals(Direction.OUT)) {
+            return outVertexId;
+        }
+        return null;
```

Comment:
Should we throw a VertexiumNotSupportedException exception here? I think I'd expect it to blow up if I passed in `Direction.BOTH`